drivers/soundwire/bus.c

```diff
@@ -1239,20 +1239,27 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 
 	sdw_modify_slave_status(slave, SDW_SLAVE_ALERT);
 
+	ret = pm_runtime_get_sync(&slave->dev);
+	if (ret < 0 && ret != -EACCES) {
+		dev_err(&slave->dev, "Failed to resume device: %d\n", ret);
+		pm_runtime_put_noidle(slave->bus->dev);
```

Comment:
just wondering if the modify_slave_status() just above needs to be addressed if the device failed to resume?

Reply:
no, the slave status is what we see on the bus, it's different from the status of the Linux device.

The English language does not have enough terms for 'device'